dyno-queues-redis/src/main/java/com/netflix/dyno/queues/redis/RedisDynoQueue.java

```diff
@@ -429,6 +429,29 @@ public boolean remove(String messageId) {
         }
     }
 
+    @Override
+    public boolean ensure(Message message) {
```

Comment:
we would need this in the v2 DynoQueues also.

Reply:
Yes I forgot to do that, I'll add that shortly.

Reply:
On second thought, let's do the V2 part once we figure out the code restructuring for V2. I've made it throw and UnsupportedOperationException for V2 now.